agent/app/agent.go

```diff
@@ -370,6 +372,7 @@ func (agent *ecsAgent) newTaskEngine(containerChangeEventStream *eventstream.Eve
 
 	// Use the values we loaded if there's no issue
 	agent.containerInstanceARN = previousContainerInstanceArn
+	agent.registrationToken = previousRegistrationToken
```

Comment:
previousRegistrationToken will never be initialized here. So the agent.registrationToken will always equal to "". It can be removed